Automattic/jetpack
# GIF Block: Fix Search Bar Input Styling

Fixes #20686

## Proposed changes:

* Ensures that the GIF block's search is correctly aligned (and full width) like the other Embed blocks.

## Jetpack product discussion


N/A.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:
Insert a GIF block in the editor, and compare the styling of the input bars. You'll also want to verify the difference when a GIF has been inserted, since it's possible to edit the GIF afterwards too. 

| Before | After |
|--------|--------|
| <img width="899" alt="Screenshot 2023-12-26 at 14 26 22" src="https://github.com/Automattic/jetpack/assets/43215253/520af810-cff8-4edf-99e1-cccf05168a82"> | <img width="949" alt="Screenshot 2023-12-26 at 14 25 55" src="https://github.com/Automattic/jetpack/assets/43215253/da413e1b-b07f-4fb7-bd22-1eef608d6d38"> |
| <img width="888" alt="Screenshot 2023-12-26 at 14 26 42" src="https://github.com/Automattic/jetpack/assets/43215253/e1112db4-cf34-4fda-9001-e40a65e23d75"> | <img width="895" alt="Screenshot 2023-12-26 at 14 25 42" src="https://github.com/Automattic/jetpack/assets/43215253/4d9a29bf-3bc7-4ce0-9fdd-8e6785772895"> |
